packages/terra-disclosure-manager/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Replaced 'size' with 'dimensions' in `disclose` API.
```

Comment:
It might be worth calling out that this was for documentation. It reads like it was a breaking change to the real API